### ISSUE TITLE
Loosing information about single/double quotes in the attribute value

### ISSUE THREAD

**Issue by grych** [philss/floki#181]
When parsing, Floki loses the information if the attribute was introduced as single or double quoted text. This is generating an issue when rendering attributes with double quotes inside.

Observe the example:
```elixir
iex> html = "<tag attr='one \"two\" three'>"
"<tag attr='one \"two\" three'>"
iex> Floki.attribute(html, "attr")
["one \"two\" three"]
```
Attribute is retrieved correctly, as a proper Elixir string. But let's try to re-render the parsed HTML:
```elixir
iex> rerendered = html |> Floki.parse() |> Floki.raw_html()
"<tag attr=\"one \"two\" three\"></tag>"
```
This is a different html than it was in the beginning. And the attribute value has changed:
```elixir
iex> Floki.attribute(rerendered, "attr")
["one "]
```

This behaviour causes this issue: grych/drab#117. Notice that using quotes inside the attribute value is [correct](https://www.w3schools.com/html/html_attributes.asp).

I understand that Floki does not store the information about single/double quotes used in the attribute, but maybe the simple workaround exists, like **if there is a double quote inside the attribute value, render it with single quote**?

**Comment by mischov** [comment]
Good point. I can't think of a better solution.

**Comment by grych** [comment]
It should do the trick, as no other escaping single or double quotes is permitted in the attribute value, as well as the value containing both single and double quote in the same time is not allowed.

**Comment by philss** [comment]
@grych good catch! I think your solution should work good. Please go ahead with a PR. Thank you!

**Comment by grych** [comment]
@philss will do!

**Comment by grych** [comment]
Fixed in #182. 
Please let me know when it comes into the release, as I need to update deps for Drab. Thanks!
